libraries/types/pbft_block/include/pbft/sync_block.hpp

```diff
@@ -22,12 +22,13 @@ class SyncBlock {
   explicit SyncBlock(bytes const& all_rlp);
 
   std::shared_ptr<PbftBlock> pbft_blk;
-  std::vector<std::shared_ptr<Vote>> cert_votes;
+  std::vector<std::shared_ptr<Vote>> cert_votes;  // These votes are the cert votes that validate previous pbft block
```

Comment:
call it then previous_block_cert_votes

Reply:
From Miho branch, I fixed in this branch